Assets/Scripts/Characters/Player/PlayerAbility.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Mirror;
+
+public abstract class PlayerAbility : NetworkBehaviour
```

Comment:
Maybe we can make player abilities non-networkbehaviours, that would also be optimization